Tools/Dev/WalletAgent/manage

```diff
@@ -149,7 +149,7 @@ configureEnvironment() {
   done
 
   # Global
-  export DEVELOPER_WORKSTATION_IP="10.255.255.175"
+  export DEVELOPER_WORKSTATION_IP="192.168.1.76"
```

Comment:
It would be better to avoid changing this in commits to prevent conflicts when other developers pull the new code.

Reply:
sure